tests/integration/component.go

```diff
@@ -338,7 +338,18 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "create", "nodejs", "--project", project, "--env", "key=value,key1=value1")...)
 			helper.CmdShouldPass("odo", append(args, "push")...)
 			helper.CmdShouldPass("odo", append(args, "delete", "--all", "-f")...)
+			files := helper.ListFilesInDir(context)
+			Expect(files).NotTo(ContainElement(".odo"))
+		})
 
+		It("creates a local nodejs component without pushing and then deletes it using --all flag", func() {
```

Comment:
"creates a local nodejs component **without pushing** and then deletes it using --all flag" -> "creates a local python component and push then deletes it using --all flag"